### ISSUE TITLE
Not able to control Zappi

### ISSUE THREAD

**Issue by Jerker74** [CJNE/ha-myenergi#221]
Component Version 0.0.20

HA version
![image](https://user-images.githubusercontent.com/91065694/188139765-9e72aba5-5bef-421a-98e5-ba47a6ec126f.png)

Logg file
![Zappi bug](https://user-images.githubusercontent.com/91065694/188139814-c47d6be3-4f3a-4c1c-a61d-3d4c0b020dd4.png)

Bug description
This stopped working some days ago. I ran component version 0.0.18 when it stopped work. Now I upgraded to 0.0.20 but there is no difference. 

I was able to control zappi. Now it seems like I set (every 60s) the status but nothing happens at zappi. Then a few seconds later the actual zappi status is read back into HA and changes the value I set. 

Bottom line: I can read the status correctly but my set procedure does not take effect. Can this relate to HA 2022.8.7?

**Comment by Ulrar** [comment]
Are you getting an error when using the drop down to select an option saying the entity ID is missing ?
It's doing that for me today, and my automation on it is now referencing 'unknown state' so something changed somewhere and broke this

**Comment by Jerker74** [comment]
Hi,

No, I don't get any error messages, everything looks correct. When I set
the state and for a short period of time it looks like (in HA) zappi
actually changed state. But within a minute the state is back to the value
which was set on the zappi box. I have searched all log files I'm familiar
with but nothing. Yesterday I tried to update the hub but no changes there
either. To me it looks like something is wrong with the messages going from
the hub to zappi.

Den tors 15 sep. 2022 kl 22:42 skrev Kevin Lemonnier <
***@***.***>:

**Comment by Ulrar** [comment]
Yeah that sounds like the hub is too far from the Zappi, or at least that's the behavior I had when mine was in the house.

Did you try moving the hub closer, or confirming if the official app is slow to apply mode changes too ?

**Comment by Jerker74** [comment]
I moved the hub closer and that worked. Thanks for the support.😊

Den fre 16 sep. 2022 08:36Jerker Andersson ***@***.***> skrev:

**Comment by Ulrar** [comment]
Great! Had the same issue a few months ago myself.
You can probably close this issue then !
